tests/plugins/nullabilityGuardPlugin.spec.ts

```diff
@@ -92,12 +92,12 @@ const types = [
   }),
   queryField('intList', {
     type: 'Int',
-    list: true,
+    list: [true],
     resolve: () => [1, 2, null],
   }),
   queryField('userList', {
     type: 'User',
-    list: true,
+    list: [true],
```

Comment:
The array syntax for `list` has always felt kind of awkward. Was thinking we could deprecate the `list` property on the object here, instead adding `list` and `nonNull` helper fns similar to graphene https://docs.graphene-python.org/en/latest/types/list-and-nonnull/, so this would become:

```ts
import { list, queryField } from '@nexus/schema'

queryField('userList', {
  type: list('User'),
  resolve: () => [null, Promise.resolve(null), null],
})
```

Haven't taken a shot at it yet, but I don't think it'd be too difficult to type / unwrap as necessary.

Reply:
Could we defer this change to another issue/PR. Agree that we have room to improve here. But I don't think this is necessary to tackle here or in order to ship nullability defaults change.

@tgriesser OK with you?

Reply:
> The array syntax for `list` has always felt kind of awkward. Was thinking we could deprecate the `list` property on the object here, instead adding `list` and `nonNull` helper fns similar to graphene https://docs.graphene-python.org/en/latest/types/list-and-nonnull/, so this would become:
> 
> ```ts
> import { list, queryField } from '@nexus/schema'
> 
> queryField('userList', {
>   type: list('User'),
>   resolve: () => [null, Promise.resolve(null), null],
> })
> ```
> 
> Haven't taken a shot at it yet, but I don't think it'd be too difficult to type / unwrap as necessary.

Why to drop it? Is it hard to keep those two options?

Reply:
@homoky we can keep it around for a bit (or forever) I suppose - just like the idea of consolidating on one way to do things when it's practical.